### ISSUE TITLE
Allow specifying commit footers

### ISSUE THREAD

**Issue by CameronDiver** [dependabot/feedback#734]
I took a look at https://github.com/dependabot/feedback/issues/443 and it appears it's not possible to add a footer to the commit message. Our CI requires a `signed-off-by` and `change-type` footer, when change type can be `patch|minor|major`.

Currently it looks like I'll have to add these manually, but will there be support for something like this?

Footers in this case are defined:
```
commit title

commit body
which can have

multiple newlines

# footers here:
signed-off-by: ...
change-type: patch
```

**Comment by rebelagentm** [comment]
👋 Thank you for submitting this!  I'm not sure at this time if support will be added for this.  We're currently swamped scaling Dependabot for GitHub, so it will, unfortunately, be some time before we can take a look at enhancements like this.

**Comment by StefKors** [comment]
Hi @rebelagentm, It's now a couple of months later, could this feature go up for another consideration?

**Comment by rebelagentm** [comment]
Hi, @StefKors!  I'm no longer working on Dependabot.  @feelepxyz could better speak to this.

**Comment by feelepxyz** [comment]
@StefKors yeah we're currently doing a bunch of work to eventually migrate everyone to config files so will consider this after we've completed this project.

**Comment by msanguineti** [comment]
> we're currently doing a bunch of work to eventually migrate everyone to config files

@feelepxyz does this mean that the Web GUI will not work anymore, or that it will be enhanced to manipulate **every** option in the config files? 

There are few options not exposed in the web GUI which we find useful (e.g. commit message header set to `chore` for development deps and to  `fix` for runtime deps)

**Comment by feelepxyz** [comment]
We’re removing the web GUI entirely and moving all configuration to config files. We’ll be providing an automated migration path that converts your existing account to the new config file format.
